packages/@vuepress/core/lib/client/root-mixins/updateMeta.js

```diff
@@ -95,7 +95,7 @@ function metaIdentifier (tag) {
  * Render meta tags
  *
  * @param {Array} meta
- * @returns {Array<string>}
+ * @returns {string}
```

Comment:
```suggestion
 * @returns {String}
```

Great catch @Mister-Hope! Just wanted to capitalize the type to be consistent with the rest of the file.